meta-cube/recipes-support/overc-system-agent/files/overc-system-agent-1.2/Overc/overc.py

```diff
@@ -68,10 +70,31 @@ def _system_upgrade(self, template, reboot, force):
                             print "*** Failed to upgrade container %s" % cn
                             print "*** Abort the system upgrade action"
                             sys.exit(self.retval)
+                        else:
+                            if self.container.is_overlay(cn) > 0:
+                                overlay_flag = 1
```

Comment:
I'm not seeing the command line way to override this operation (as was requested in the previous review).